### PR TITLE
🚨🚨🐛Source Hubspot: update marketing_emails stream schema

### PR DESCRIPTION
**Pull request by askarpets**
## What
https://github.com/airbytehq/oncall/issues/3866

## How
Update data types in the `marketing_emails` schema and extend it with additional fields.
Also, the following schemas were extended, but this update is not a breaking change since only new fields were added:

- `contact_lists`
- `engagements`
- `forms`
- `workflows`

## Recommended reading order
1. `marketing_emails.json`
2. `contact_lists.json`
3. `engagements.json`
4. `forms.json`
5. `workflows.json`

## 🚨 User Impact 🚨
This is a breaking change since several data types in the schema were changed.

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-hubspot/metadata.yaml [hunk 1]
```diff
@@ -30,6 +30,11 @@ data:
           This version eliminates the Property History stream in favor of creating 3 different streams, Contacts, Companies, and Deals, which can now all fetch their property history.
           It will affect only users who use Property History stream, who will need to fix schema conflicts and sync Contacts Property History stream instead of Property History.
         upgradeDeadline: 2024-01-15
+      3.0.0:
+        message: >-
+          This version brings extended schema with data types changes for the Marketing Emails stream.
+          Users will need to refresh it and reset this stream after upgrading.
+        upgradeDeadline: 2024-02-12
```
Comment (artem1205) [inline airbyte-integrations/connectors/source-hubspot/metadata.yaml:37]: please add scopedImpact section

Comment (askarpets) [reply airbyte-integrations/connectors/source-hubspot/metadata.yaml:37]: Added, thanks!

**Review by tolik0 — APPROVED** [review]

File: docs/integrations/sources/hubspot-migrations.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # HubSpot Migration Guide
 
+## Upgrading to 3.0.0
+
+:::note
+This change is only breaking if you are syncing the Marketing Emails stream.
+:::
+
+This update brings extended schema with data types changes for the Marketing Emails stream.
```
Comment (katmarkham) [inline docs/integrations/sources/hubspot-migrations.md:9]: This update brings extended schema with data type changes for the Marketing Emails stream.

Comment (katmarkham) [reply docs/integrations/sources/hubspot-migrations.md:9]: please give them the appropriate migration guide steps: https://docs.google.com/document/d/1V3ybA24Lj_fy7IQwsVVzSxs46qGOsUydwlAQEYSQfcE/edit?pli=1

**Review by katmarkham — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/hubspot-migrations.md (hunk 2 above)
Comment (askarpets) [reply docs/integrations/sources/hubspot-migrations.md]: Updated

**Review by katmarkham — APPROVED** [review]

**Pull request merged**